packages/shopify-codemod/test/fixtures/constructor-literal-assignment-to-class-property/complex-literals.output.js

```diff
@@ -0,0 +1,11 @@
+class Foo {
+  foo = {bar: 12, baz: true};
+  bar = {[baz]: 'something'};
```

Comment:
does this make the eslint rule fail if we don't put it as a class property? because it's pretty confusing to have to intersperse the contructor and these lines to be able to make sense of it (like, there's no easy way to see where baz came from)